### PR TITLE
Event Dashboard: Logos and Columns scaled properly #3091

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**

### CODE REVIEW

**Comment by Himanshi-Khandelwal** [comment]
@niranjan94 there occured some problem in that branch.

**Comment by niranjan94** [comment]
@Himanshi-Khandelwal please check your commits. Squash them. (https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/)

**Comment by Himanshi-Khandelwal** [comment]
@niranjan94 done!!

**Comment by niranjan94** [comment]
@Himanshi-Khandelwal we have file conflicts here. Needs resolution. 

Please ensure your pull requests are good and easy to review with minimal changes. This PR for example has been open for almost a week. And it's just a simple fix. 

Try to see how others PR are and what are the changes that were requested in them before they were approved for merge. Learn from your mistakes and the mistakes of other contributors too. 

Making a perfect PR maybe not be easy right now since you're very new to the project.. And that's completely alright , but atleast ensure all the non-technical aspects of the PR such as commit messages, conflicts, squashing etc proper without a reviewer having to request them. That way, it's easier for the reviewer too to quickly review the code and approve.

File: app/templates/gentelella/users/events/_table.html [hunk 1]
```diff
@@ -42,7 +46,11 @@
                 {% endfor %}
 
                 <tr>
+
+                    <td><a href="{{ url_for('events.details_view', event_id=event.id) }}">  <img src="{{ event.logo }}"  onerror="imgError(this); " style=" width:70px;"><p style ="padding:8px;">{{ event.name }}</p></a></td>
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/_table.html]: The line is duplicated here. And only thing that is supposed to have changed in this PR is this line ... How come there are so many changes here ? Please check

**Comment by Himanshi-Khandelwal** [comment]
@niranjan94  I don't know there is some problem coming.. not able to squash. tried a lot ( did google search also) . don't know whats the problem . so I am deleting this fork and cloning once again.

**Pull request closed without merging**